python/core/auto_generated/qgsfeaturerequest.sip.in

```diff
@@ -405,6 +405,8 @@ will use the same CRS as the source layer/provider.
 .. versionadded:: 3.22
 %End
 
+    std::shared_ptr< QgsGeometryEngine > referenceGeometryEngine() const;
```

Comment:
This shouldn't be exposed to sip... it's the sort of API we should reserve the ability to revise in future

Reply:
(also keeping in mind that if we do end up storing prepared instances in QgsGeometryPrivate then this method will no longer be required)

Reply:
Hidden in next commit